src/app/component/ProductSort/ProductSort.component.js

```diff
@@ -68,7 +42,21 @@ class ProductSort extends Component {
     }
 
     render() {
-        const { value, sortFields, sortDirection } = this.props;
+        const {
```

Comment:
What this functional does?

Reply:
This is the custom select functionality. I guess my-account-frontend did not have it yet.